src/lib/_imports/elements/table--basic.css

```
@@ -3,18 +3,20 @@
 table,
 :--s-paragraph-table {
   --font-size: var(--global-font-size--small, 12px);
-  --cell-pad-vert: 0.6667em; /* 8px if font-size is 12px */
-  --cell-pad-horz: 1.3333em; /* 16px if font-size is 12px */
-  --cell-line-height: 1.1;
+  --cell-pad-vert: 1em;
+  --cell-pad-horz: 2em;
 
   /* To prevent borders from being hidden */
   /* SEE: TABLE WITH A <thead> */
-  border-collapse: separate; /* overwrite Bootstrap */
-  border-spacing: 0;
+  border-collapse: separate;
+  border-spacing: 0; /* overwrite Bootstrap */
```

To revert accidental movement of comment:

```suggestion
  border-collapse: separate; /* overwrite Bootstrap */
  border-spacing: 0;
```